types/src/lib.rs

```diff
@@ -66,3 +66,5 @@ impl NetworkParams for DevnetParams {
 
 /// Space-race specific fork
 pub const UPGRADE_SMOKE_HEIGHT: i64 = 51000;
+pub const UPGRADE_IGNITION_HEIGHT: i64 = -2;
+pub const UPGRADE_LIFTOFF_HEIGHT: i64 = -3;
```

Comment:
These are fake params given to new networks, if you're going to use the fork upgrade logic, this should be correct.